From e447962ea8b09d638c8ff928eab3c062721f18fa Mon Sep 17 00:00:00 2001
From: Robby Cai <robby.cai@nxp.com>
Date: Fri, 13 May 2016 10:03:35 +0800
Subject: [PATCH 1067/5242] MLK-12811-3 ARM: dts: add new dts file for EPDC on
 i.MX6ULL DDR3 VAL board

commit  f7e89799455fff119a152a428b367b415dff01dd from
https://source.codeaurora.org/external/imx/linux-imx.git

EPDC has pin conflict with FEC2, LCDIF.

Signed-off-by: Robby Cai <robby.cai@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm/boot/dts/Makefile                         |    1 +
 arch/arm/boot/dts/imx6ull-14x14-ddr3-arm2-epdc.dts |   25 ++++++++++++++++++++
 2 files changed, 26 insertions(+)
 create mode 100644 arch/arm/boot/dts/imx6ull-14x14-ddr3-arm2-epdc.dts

diff --git a/arch/arm/boot/dts/Makefile b/arch/arm/boot/dts/Makefile
index a6aec03..0347a92 100644
--- a/arch/arm/boot/dts/Makefile
+++ b/arch/arm/boot/dts/Makefile
@@ -604,6 +604,7 @@ dtb-$(CONFIG_SOC_IMX6ULL) += \
 	imx6ull-14x14-ddr3-arm2-cs42888.dtb \
 	imx6ull-14x14-ddr3-arm2-ecspi.dtb \
 	imx6ull-14x14-ddr3-arm2-emmc.dtb \
+	imx6ull-14x14-ddr3-arm2-epdc.dtb \
 	imx6ull-14x14-ddr3-arm2-flexcan2.dtb \
 	imx6ull-14x14-ddr3-arm2-lcdif.dtb \
 	imx6ull-14x14-ddr3-arm2-ldo.dtb \
diff --git a/arch/arm/boot/dts/imx6ull-14x14-ddr3-arm2-epdc.dts b/arch/arm/boot/dts/imx6ull-14x14-ddr3-arm2-epdc.dts
new file mode 100644
index 0000000..c476f44
--- /dev/null
+++ b/arch/arm/boot/dts/imx6ull-14x14-ddr3-arm2-epdc.dts
@@ -0,0 +1,25 @@
+/*
+ * Copyright (C) 2016 Freescale Semiconductor, Inc.
+ *
+ * This program is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 2 as
+ * published by the Free Software Foundation.
+ */
+
+#include "imx6ull-14x14-ddr3-arm2.dts"
+
+&epdc {
+        status = "okay";
+};
+
+&fec2 {
+        status = "disabled";
+};
+
+&lcdif {
+        status = "disabled";
+};
+
+&max17135 {
+        status = "okay";
+};
-- 
1.7.9.5

